cloud/types.go

```diff
@@ -299,7 +323,7 @@ type (
 		ChecksSuccessCount    int        `json:"checks_success_count"`
 		CreatedAt             *time.Time `json:"created_at,omitempty"`
 		UpdatedAt             *time.Time `json:"updated_at,omitempty"`
-		PushedAt              *time.Time `json:"pushed_at,omitempty"`
+		PushedAt              any        `json:"pushed_at,omitempty"`
```

Comment:
Why don't you use here `int64` instead and put the others as time.Unix()?

Reply:
I didn't want to change the behavior of Github and Gitlab, trying to minimize regressions in the other platforms but if you say sending an integer for all CIs is okay, I will do it.